website/docs/d/alert_configuration.html.markdown

```diff
@@ -66,35 +100,35 @@ In addition to all arguments above, the following attributes are exported:
 * `enabled` - If set to true, the alert configuration is enabled. If enabled is not exported it is set to false.
 * `event_type` - The type of event that will trigger an alert.
   Alert type. Possible values:
-	 - Host 	
+	 - Host
```

Comment:
event_type was massively updated in the docs - https://docs.atlas.mongodb.com/reference/api/alert-configurations-create-config/ so to close both issues we'll need to address this.   The updated values can be found in the "Condition Tab" under eventTypeName.   We don't need the descriptions here but I would perhaps list the legal values and put a link to the underlying API for all the explanations as they are subject to change.

Reply:
It makes sense, I think this is better to remove them and delegate the error handling to MongoDB server, thanks